vlttng/venv.py

```diff
@@ -440,7 +453,9 @@ def _create_project_instructions_lttng_tools(self, project):
     def _create_project_instructions_lttng_ust(self, project):
         instructions = self._create_project_instructions_generic_autotools(project)
         instructions.add_env = {
-            'CLASSPATH': self._paths.log4j_jar,
+            'CLASSPATH': "{}".format(
+                ":".join([self._paths.log4j_jar] + self._paths.log4j2_jars)
```

Comment:
Single quotes.

This whole thing may be just after `CLASSPATH:` on the same line.

Reply:
Done.